requirements/requirements.in

```diff
@@ -0,0 +1,5 @@
+# Core dependencies for documentation builds
+
+sphinx                              # Documentation builder
+edx-sphinx-theme                    # edX theme for the generated documentation
+pip-tools                           # Python requirements file management utilities
```

Comment:
Add a [trailing newline](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).